.travis.yml

```diff
@@ -17,6 +17,8 @@ gemfile:
 
 matrix:
   exclude:
+    - rvm: 2.2.3
+      gemfile: Gemfile.rails5
```

Comment:
Please undo these changes and rebase (and squash) on latest master for a proper CI run. The issues with bundler have been fixed in #217 :)